cni/telemetry/service/telemetrymain.go

```diff
@@ -127,7 +127,7 @@ func main() {
 
 	logger.Info("Config path", zap.String("path", configPath))
 
-	config, err = telemetry.ReadConfigFile(configPath)
+	config, err = tb.ReadConfigFile(configPath)
```

Comment:
I don't think this is correct. Did we create a new telemetry buffer to use the ReadConfigFile ?